content/docs/onboarding/new_joiners/checklist.md

```diff
@@ -81,3 +81,4 @@ If anything is unclear, please figure it out by e.g. asking your buddies, and th
 ## Ongoing activity sign-up
 
 - [ ] "Tech Talks" (Tuesdays 12:30pm), link on: https://github.com/alan-turing-institute/DataScienceSkills/wiki/Lunchtime-Tech-Talks
+- [ ] Read up about and optionally join the [Society of Research Software Engineering](https://society-rse.org/), including the [Slack workspace](https://ukrse.slack.com/join/signup)
```

Comment:
I'd suggest a stronger stance on joining, given the benefits and conference discount. Suggest 'optionally' -> 'you are recommended to'

Reply:
Thanks for the suggestion; I've made this change.